package.json

```diff
@@ -90,20 +70,25 @@
   },
   "prettier": {
     "singleQuote": true,
-    "trailingComma": "all"
+    "trailingComma": "all",
+    "arrowParens": "avoid"
   },
   "scripts": {
-    "build": "npm run lint && npm run build:webpack",
+    "build": "npm run lint && npm run build:typescript && npm run build:webpack",
+    "build:typescript": "tsc",
     "build:webpack": "webpack --config _develop/webpack.config.js; rm dist/quill.core dist/quill.bubble dist/quill.snow",
     "build:release": "./_develop/scripts/release.sh",
     "develop": "npm run start",
-    "lint": "eslint .",
+    "lint": "npm run lint:eslint && npm run lint:tsc",
+    "lint:eslint": "eslint .",
+    "lint:tsc": "tsc --noEmit --skipLibCheck",
+    "prepare": "npm run build",
```

Comment:
When Quill is installed as a git dependency, the `prepare` script will be called by npm to build js files. It won't be called when it's installed as a normal npm package where files have already been built before publishing.